lib/insights/api/common/version.rb

```diff
@@ -1,7 +1,7 @@
 module Insights
   module API
     module Common
-      VERSION = "5.0.1".freeze
+      VERSION = "5.0.2".freeze
```

Comment:
@hsong-rh I think this is a breaking change, it should be 6.0.0 since its using the latest version of Rails and we could have broken things.